mkdocs.yml

```diff
@@ -0,0 +1,121 @@
+              - Skip Traversal of Specific Files: examples/standalone/skip/files.md
+              - Skip Update of Vulnerability Database: examples/standalone/skip/update-vuln-db.md
+          - Specify:
+              - Specify Cache Backend: examples/standalone/specify/cache-backend.md
+              - Specify Cache Directory: examples/standalone/specify/cache-directory.md
+              - Specify Exit Code: examples/standalone/specify/exit-code.md
+          - Clear Caches: examples/standalone/clear-caches.md
+          - Embed in Dockerfile: examples/standalone/embed-in-dockerfile.md
+          - Only Download Vulnerability Database: examples/standalone/only-download-vuln-db.md
+          - Reset: examples/standalone/reset.md
+          - Use Lightweight Database: examples/standalone/lightweight-db.md
+      - Client/Server: examples/client-server.md
+  - Continuous Integration (CI):
+      - Overview: continuous-integration/index.md
+      - AWS CodePipeline: continuous-integration/aws-codepipeline.md
+      - AWS Security Hub: continuous-integration/aws-security-hub.md
+      - CircleCI: continuous-integration/circleci.md
+      - GitHub Actions: continuous-integration/github-actions.md
+      - GitLab CI: continuous-integration/gitlab-ci.md
+      - Travis CI: continuous-integration/travis-ci.md
+      - Authorization for Private Docker Registry:
+          - Overview: continuous-integration/private/index.md
+          - AWS ECR (Elastic Container Registry): continuous-integration/private/ecr.md
+          - Docker Hub: continuous-integration/private/docker-hub.md
+          - GCR (Google Container Registry): continuous-integration/private/gcr.md
+          - Self-Hosted: continuous-integration/private/self.md
+  - Vulnerability Detection:
+      - Overview: vuln-detection/index.md
+      - OS Packages: vuln-detection/os.md
+      - Application Dependencies: vuln-detection/library.md
+      - Image TAR Format: vuln-detection/image-tar-format.md
+      - Data Sources: vuln-detection/data-source.md
+  - Air-Gapped Environment: air-gap.md
+  - Comparison with Other Scanners:
+      - Overview: comparison/index.md
+      - Anchore Engine: comparison/anchore.md
+      - Clair: comparison/clair.md
+      - Quay, Docker Hub, GCR: comparison/quay_dockerhub_gcr.md
+  - Usage:
+      - Overview: usage/index.md
+      - Image: usage/image.md
+      - Client: usage/client.md
+      - Server: usage/server.md
+  - Migration:
+      - Overview: migration/index.md
+      - Binary: migration/binary.md
+      - Debian/Ubuntu: migration/debian_ubuntu.md
+      - Homebrew: migration/homebrew.md
+      - RHEL/CentOS: migration/rhel_centos.md
+  - FAQ:
+      - Overview: faq/index.md
+      - Homebrew: faq/homebrew.md
+      - Others: faq/others.md
+  - Credits: credit.md
+
+theme:
+    name: material
+    language: 'en'
+    logo: imgs/logo.png
+
+markdown_extensions:
+  - pymdownx.highlight
+  - pymdownx.superfences
+  - admonition
+
```

Comment:
CMIIW, but [this](https://github.com/aquasecurity/starboard/blob/9c23ea820a5b52c8824d3ec4e1ec7918a8edf934/mkdocs.yml#L55-L67) looks like it is generated during deployment.